05-cond.md

```diff
@@ -65,18 +65,17 @@ which is short for "else if".
 This makes it simple to write a function that returns the sign of a number:
 
 ~~~ {.python}
-def sign(num):
-    if num > 0:
-        return 1
-    elif num == 0:
-        return 0
-    else:
-        return -1
-
-print 'sign of -3:', sign(-3)
+num = -3
+
+if num > 0:
+    print num,"is +ve"
```

Comment:
Hey,

I think that either "positive"  or "+"  (mathematics is meant to be a
universal language after all) would be fine. I agree with the comments on
the use of +ve as a contraction.

I am happy for the suggested change to be committed.

Ewan
On 5 May 2015 01:50, "Ivan Gonzalez" <notifications@github.com> wrote:

> In 05-cond.md
> <https://github.com/swcarpentry/python-novice-inflammation/pull/77#discussion_r29595611>
> :
>
> > @@ -65,18 +65,17 @@ which is short for "else if".
> >  This makes it simple to write a function that returns the sign of a number:
> >
> >  ~~~ {.python}
> > -def sign(num):
> > -    if num > 0:
> > -        return 1
> > -    elif num == 0:
> > -        return 0
> > -    else:
> > -        return -1
> > -
> > -print 'sign of -3:', sign(-3)
> > +num = -3
> > +
> > +if num > 0:
> > +    print num,"is +ve"
>
> Yep.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swcarpentry/python-novice-inflammation/pull/77/files#r29595611>
> .
>